query-connector/src/app/query/components/searchForm/SearchForm.tsx

```diff
@@ -74,7 +72,7 @@ const SearchForm: React.FC<SearchFormProps> = ({
         setMRN(data.MRN);
         setPhone(data.Phone);
         setFhirServer(data.FhirServer as FHIR_SERVERS);
-        setUseCase(data.UseCase as USE_CASES);
+        // setUseCase(data.UseCase as USE_CASES);
```

Comment:
Does this suggest that we can clean up the demo data array setup that we have going on in the constants file and just default to the one patient? I didn't bother with the other PR since it was getting kinda big already, but would love to delete those constants if we don't need to do matching by use case / can just autofill with the one fixture we've created

If so, might worth doing here / in a followup PR would be great!

Reply:
yeah, I wasn't sure how much we wanted to essentially toss. at one point, I was just importing `hyperLuckyPatient` to directly fill these, but cleaning up / refining this aspect might make more sense for a followup ticket.

Reply:
Actually it looks like doing this makes the default on the search form disabled since a use case is needed to do the query. You can fix this by picking an option, but the fact that it renders with "cancer" and still be disabled isn't great. We can turn this on for now and address in the follow up
<img width="735" alt="Screenshot 2024-10-17 at 8 53 19 AM" src="https://github.com/user-attachments/assets/f0fcbda8-9e50-4733-9f40-f996d422ef89">

Reply:
ack, yeah. I added `Select query` as the default to the dropdown, since that's what's in the figma, which gets around this.